src/widgets/setupdialog.cpp

```diff
@@ -230,6 +230,8 @@ bool SetupDialog::validateAdvancedSetupPage()
         emit getTarsnapVersion(_tarsnapDir);
 
     _ui.advancedPageProceedButton->setEnabled(result);
+    if (result)
+	    _ui.advancedPageProceedButton->setFocus();
```

Comment:
My apologies, my change would have been horrible.  I've fixed it locally, but I want to do more testing of the behaviour on that page to make sure I haven't made another mistake, before updating this PR.